### PR TITLE
Fix installation instructions for macOS

### PR DESCRIPTION
**Pull request by ptormene**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,11 +24,17 @@ This plugin has been created by the [GEM Foundation](http://www.globalquakemodel
 This plugin runs with [QGIS 3.0 and above](http://qgis.org/en/site/forusers/alldownloads.html)
 and above.
 
-On Microsoft Windows and Mac OS X, QGIS includes all the software dependencies needed by the plugin.
+On Microsoft Windows, QGIS includes all the software dependencies needed by the plugin.
 
-On Linux, please make sure that [Scipy](https://www.scipy.org/install.html) and
+On macOs and Linux, please make sure that [Scipy](https://www.scipy.org/install.html) and
 [Matplotlib](https://matplotlib.org/users/installing.html) with Qt4 backend are installed.
```
Comment (micheles) [inline README.md]: I think the correct spelling is `macOS`.

File: README.md [hunk 2]
```diff
@@ -24,11 +24,17 @@ This plugin has been created by the [GEM Foundation](http://www.globalquakemodel
...
```
Comment (daniviga) [inline README.md]: Qt5 not Qt4

Comment (daniviga) [inline README.md]: Qt5 not Qt4

**Comment by micheles** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (ptormene) [reply README.md]: Yes, thanks, I actually wanted to type macOS...

**Pull request merged**